lib/metasploit/framework/login_scanner/pfsense.rb

```diff
@@ -0,0 +1,116 @@
+require 'metasploit/framework/login_scanner/http'
+
+module Metasploit
+  module Framework
+    module LoginScanner
+
+      # This is the LoginScanner class for dealing with Netgate pfSense instances.
+      # It is responsible for taking a single target, and a list of credentials
+      # and attempting them. It then saves the results.
+      class PfSense < HTTP
+        LOGIN_ENDPOINT = 'index.php'
+
+        # Checks if the target is pfSense. The login module should call this.
+        #
+        # @return [Boolean, String] FalseClass if target is pfSense, otherwise String
```

Comment:
I haven't worked on too many login scanners so please excuse my ignorance - but why does this return `false` if the target _is_ pfSense, that seems unintuitive to me. Also I don't this this being directly called by the `pfsense_login.rb` login module, is that intentional?

Reply:
Good shout! This is the _expected_ way to use this API 😅 This API is wired up this way in Pro 😞

Reply:
Thanks so much for the clarification!!